vite.config.ts

```diff
@@ -10,5 +10,8 @@ export default defineConfig({
     alias: {
       '@': fileURLToPath(new URL('./src', import.meta.url))
     }
+  },
+  define: {
+    UI_VERSION: JSON.stringify(process.env.npm_package_version)
```

Comment:
I guess it's ok, but I expected `process.env.npm_package_version` to be a string, not a json object :man_shrugging:

Reply:
Done, another leftover test :grimacing: